test/test_create_model.py

```diff
@@ -0,0 +1,32 @@
+from training.creating_dataset import create_dataframe
+from training.create_model import create_model
+from training.splitting_dataset import get_random_split_results
+from sklearn.linear_model import RidgeCV
+from sklearn.ensemble import RandomForestRegressor
```

Comment:
Black to tak posortował? W sensie najpierw importy z projektu, a potem ze Scikit-learn'a? Bo to generalnie nie przestrzega kolejności importów z PEP8. Można ewentualnie dodać isort do pre-commit hooków, w trybie `black`, wtedy już na pewno będzie ładnie posortowane